HW2/P3/driver.py

```diff
@@ -22,14 +22,16 @@ def make_coords(center=(-0.575 - 0.575j),
 
     x = np.linspace(start=(-width / 2), stop=(width / 2), num=count)
     xx = center + (x + 1j * x[:, np.newaxis]).astype(np.complex64)
-    return xx, np.zeros_like(xx, dtype=np.uint32)
-
+    #return xx, np.zeros_like(xx, dtype=np.uint32)
+    return xx, np.zeros_like(xx, dtype=np.float32)
```

Comment:
it is in fact necessary for that the argument passed into the next function will have float type.